app/src/utils/MadLibs.ts

```diff
@@ -8,18 +8,21 @@ export type PhraseSelections = Record<number, number>;
 export type PhraseSegment = string | Record<number, string>;
 
 export interface MadLib {
+  readonly index: number;
```

Comment:
This could probably benefit from a comment or clearer name.

Also, do you think we should change the mad lib list to a map, and have it keyed by a mad lib id rather than index? This would break if someone switched the order of the mad libs in the list below. But I'm not sure if this needs to be a number or something.

Reply:
Yeah that did occur to me... I left it this way because I wanted to access items by ID easily but I agree this is not great. I think this was in the realm of that refactor we were talking about so feel free to do whatever I'm just put a TODO for now :)